code/modules/mob/living/carbon/human/species.dm

```diff
@@ -86,6 +87,20 @@
 	..()
 
 
+/proc/generate_selectable_species()
+	for(var/I in subtypesof(/datum/species))
+		var/datum/species/S = new I
```

Comment:
I kinda dislike creating a new instance of the species just to check if it's eligible

Reply:
Yeah you can do this with intial() trickery

Reply:
lummox give us static procs grr

Reply:
yea, sadly there's no other way if you want special checks like holidays